app/src/androidTest/java/org/simple/clinic/bloodsugar/BloodSugarRepositoryAndroidTest.kt

```diff
@@ -193,7 +193,7 @@ class BloodSugarRepositoryAndroidTest {
   fun updating_a_blood_sugar_should_update_it_correctly() {
     // given
     val durationToAdvanceBy = Duration.ofMinutes(15L)
-    val bloodSugarReading = BloodSugarReading(126.toFloat(), Random)
+    val bloodSugarReading = BloodSugarReading(126.toString(), Random)
```

Comment:
Instead of using `toString()`, please pass the value as String itself.
Please follow this everywhere.